addons/tmf_loadouts/loadouts/us_rangers_2010.hpp

```diff
@@ -0,0 +1,610 @@
+// If you want to base a loadout on an existing one, this repository contains them all:
+// https://github.com/ARCOMM/arc_misc/tree/master/addons/tmf_loadouts
+
+//Dependencies:
+//		* CUP Units
+//		* CUP Weapons
+//		* Community Factions Project
+//		* FIR AWS(AirWeaponSystem)
+//		* ACE
+
+//	Reference: https://www.battleorder.org/us-stryker-platoon-2019 (for general formation, not for equipment)
+
+//	Created by: AChesheireCat
+
+class baseMan {// Weaponless baseclass
+	displayName = "Unarmed";
+	// All randomized.
+	uniform[] = {
+		"CFP_75th_CRYE_V3_Sleeved",
+		"CFP_75th_CRYE_V4_Sleeved",
+		"CFP_75th_CRYE_V3_Full",
+		"CFP_75th_CRYE_V4_Full"
+	};
+	vest[] = {"CFP_HV5_Ciras_Olive"};
+	backpack[] = {"CFP_Camelbak_Mule_RngrGrn"};
+	headgear[] = {};
+	goggles[] = {"default"};
+	hmd[] = {"CUP_NVG_PVS15_black"};
+	// Leave empty to remove all. "Default" > leave original item.
+	faces[] = {};
+	// Leave empty to not change faces.
+	insignias[] = {};
+	// Leave empty to not change insignias
+
+	// All randomized
+	primaryWeapon[] = {};
+	scope[] = {};
+	bipod[] = {};
+	attachment[] = {};
+	silencer[] = {};
+	// Leave empty to remove all. "Default" for primaryWeapon > leave original weapon.
+
+	// Only *Weapons[] arrays are randomized
+	secondaryWeapon[] = {};
+	secondaryAttachments[] = {};
+	sidearmWeapon[] = {};
+	sidearmAttachments[] = {};
+	// Leave empty to remove all. "Default" for secondaryWeapon or sidearmWeapon > leave original weapon.
+
+	// These are added to the uniform or vest
+	magazines[] = {};
+	items[] = { MEDICAL_R ,"ACE_IR_Strobe_Item","ACE_Flashlight_XL50","ACE_SpraypaintGreen",LIST_2("ACE_CableTie"),LIST_2("ACE_Chemlight_HiGreen")};
+	// These are added directly into their respective slots
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemWatch",
+		"ItemRadio"
+	};
+
+	// These are put into the backpack
+	backpackItems[] = {"CFP_75th_Beanie_Grn"};
+
+	// Unit traits, see https://community.bistudio.com/wiki/setUnitTrait
+	traits[] = {};
+
+	// This is executed after unit init is complete. argument: _this = _unit.
+	code = "";
+};
+
+class r : baseMan {
+	displayName = "Rifleman";
+	headgear[] = {
+		LIST_2("CUP_H_CZ_Helmet09"),
+		LIST_2("CUP_H_CZ_Helmet10"),
+		"H_HelmetB_camo"
+	};
+	primaryWeapon[] = {
+		"CUP_arifle_M4A3_black",
+		"CUP_arifle_M4A1_black",
+		"CUP_arifle_Mk16_STD_AFG"
+	};
+	scope[] = {
+		LIST_2("cup_optic_eotech553_coyote"),
+		"cup_optic_compm2_low"
+	};
+	silencer[] = {
+		"cup_muzzle_snds_m16_coyote",
+		"cup_muzzle_snds_m16"
+	};
+	attachment[] = {
+		"cup_acc_anpeq_15_top_flashlight_tan_l",
+		"cup_acc_anpeq_15_tan_top",
+		"cup_acc_anpeq_15"
+	};
+	magazines[] = {
+		LIST_7("CUP_30Rnd_556x45_PMAG_QP"),
+		LIST_2("CUP_30Rnd_556x45_Emag_Tracer_Red"),

```

Comment:
PMAG/EMAG mixed?

Reply:
Yes - idea was to provide some visually identifiable differences between the tracer magazines and the normal magazines. I can simplify it if needed.